packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -46,12 +47,25 @@ const useStyles = makeStyles(theme => ({
     '&.Mui-expanded': {
       borderBottom: `1px solid ${theme.palette.divider}`
     }
+  },
+  accordionHeader: {
+    lineHeight: 2.5
   }
 }))
 
-export default function ResultSet({ data }) {
+export default function ResultSet({ data, onChangeSelection }) {
   const classes = useStyles()
 
+  const [isSelected, setIsSelected] = React.useState(false)
```

Comment:
I am using this to avoid having a large part of the document just importing stuff. I will make this requested change.

Reply:
IMO is a thing that has to be done in this merge, for 2 reasons:
- importing React is a huge operation, and also sometimes breaks tree-shaking;
- since the introduction of the new JSX runtime with React 17, it is discouraged

Reply:
If you don't want to make by yourself, you can use an automatic codemod:
https://github.com/reactjs/react-codemod#update-react-imports

Reply:
@ilteoood as I said, I will make this change. Is not in the PR yet.

Reply:
Ok, so it's not correct to resolve the conversation if the comment has not been already addressed.
Otherwise, the next reviewer will came with the same comment 😂

Reply:
I was addressed but I didn't request again the review 😄